### ISSUE TITLE
Code problem

### ISSUE THREAD

**Issue by rongtongxueya** [mpskex/AttentiveContrastiveLearningNetwork#1]
“ File "/home/pqt/rcx/AttentiveContrastiveLearningNetwork-master/fgvc_cl/networks/__init__.py", line 2, in <module>
    from . import APINet, model_zoo, AttentiveContrative, BareBackbone
ImportError: cannot import name 'APINet' from partially initialized module 'fgvc_cl.networks' (most likely due to a circular import) ”

May I ask why there is this problem, it has taken a long time to solve this problem

**Comment by mpskex** [comment]
Do you have any scripts using this library? This may caused by your working directory.

**Comment by rongtongxueya** [comment]
No, I just downloaded your code and wanted to run it to see the result and then read it carefully, but this problem occurs when I run your code

**Comment by mpskex** [comment]
> No, I just downloaded your code and wanted to run it to see the result and then read it carefully, but this problem occurs when I run your code

Please provide your script and full traceback of your error message. The information you provided here does not help us to locate your issue.

**Comment by JACYI** [comment]
> “ File "/home/pqt/rcx/AttentiveContrastiveLearningNetwork-master/fgvc_cl/networks/**init**.py", line 2, in from . import APINet, model_zoo, AttentiveContrative, BareBackbone ImportError: cannot import name 'APINet' from partially initialized module 'fgvc_cl.networks' (most likely due to a circular import) ”
> 
> May I ask why there is this problem, it has taken a long time to solve this problem

delete the 'APINet' in `./fgvc_cl/networks/__init__.py`
```
from . import APINet, model_zoo, AttentiveContrative, BareBackbone
```
↓
```
from . import model_zoo, AttentiveContrative, BareBackbone
```

**Comment by mpskex** [comment]
Does this fix the issue? I will help to merge this into master
